src/ape/managers/query.py

```diff
@@ -30,12 +58,13 @@ def engines(self) -> Dict[str, QueryAPI]:
             dict[str, :class:`~ape.api.query.QueryAPI`]
         """
 
-        engines = {}
+        engines = {"__default__": DefaultQueryProvider()}
```

Comment:
I think this will solve the typing issue below (e.g. the typing is more constrained here than it should be)
```suggestion
        engines: Dict[str, QueryAPI] = {"__default__": DefaultQueryProvider()}
```